### PR TITLE
Update repo toolset

### PR DESCRIPTION
**Pull request by cdmihai**

### CODE REVIEW

**Comment by cdmihai** [comment]
Waiting on https://github.com/dotnet/dotnet-ci/pull/992 to get CI jobs

**Comment by cdmihai** [comment]
@dotnet-bot test this please

**Comment by cdmihai** [comment]
@dotnet-bot test this please

**Comment by cdmihai** [comment]
The vsix has the same content as the previous one.

**Comment by cdmihai** [comment]
@dotnet-bot test this please

**Comment by cdmihai** [comment]
@mmitche I forgot, what do I have to do to make this PR trigger the CI jobs after the target branch got added to repolist?

File: src/Directory.BeforeCommon.targets [hunk 1]
```diff
@@ -4,11 +4,7 @@
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(PlatformTarget)' == 'x64'">
-    <OutputPath>$(OutputPath)$(Platform)\$(TargetFramework.ToLowerInvariant())\</OutputPath>
-    <IntermediateOutputPath>$(IntermediateOutputPath)$(Platform)\$(TargetFramework.ToLowerInvariant())\</IntermediateOutputPath>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(PlatformTarget)' != 'x64'">
+  <PropertyGroup>
     <OutputPath>$(OutputPath)$(TargetFramework.ToLowerInvariant())\</OutputPath>
```
Comment (rainersigwald) [inline src/Directory.BeforeCommon.targets]: how does this not conflict for the x64 version now?

File: build/Versions.props [hunk 2]
```diff
@@ -28,19 +28,14 @@
 
   <!-- Repo Toolset Features -->
   <PropertyGroup>
-    <UsingToolMicrosoftNetCompilers>false</UsingToolMicrosoftNetCompilers>
-
     <UsingToolVSSDK>true</UsingToolVSSDK>
-    <!-- Tell repo toolset not to define the GetBuildVersion target, as it would conflict with NerdBank.GitVersioning -->
-    <DefineGetBuildVersionTargetForVsix Condition="'$(DisableNerdbankVersioning)' != 'true'">false</DefineGetBuildVersionTargetForVsix>
```
Comment (rainersigwald) [inline build/Versions.props]: How is the conflict handled now?

**Comment by rainersigwald** [comment]
Ran https://ci2.dot.net/job/Microsoft_msbuild/job/vs15.7_sourcebuild/job/generator/ manually, generating jobs.

@dotnet-bot test this please

**Comment by mmitche** [comment]
@rainersigwald The generation step needs an initial commit to start (or be triggered manually)

**Comment by rainersigwald** [comment]
@mmitche Good to know, thanks.

Also @cdmihai I assume you want to light up the RHEL job for this branch?

**Comment by mmitche** [comment]
I think we have one...I think our DevEx queue has this.

`Utilities.setMachineAffinity('Windows.10.Amd64.ClientRS3.DevEx.Open')`

**Comment by cdmihai** [comment]
@dotnet-bot test this please

**Comment by cdmihai** [comment]
@dotnet-bot test Ubuntu14.04 Build for CoreCLR please
Peculiar node failure https://ci2.dot.net/job/Microsoft_msbuild/job/vs15.7_sourcebuild/job/innerloop_Ubuntu14.04_CoreCLR_prtest/11/consoleFull#118724219379494335-f7bd-47d0-8771-8661e00c2db2

**Comment by cdmihai** [comment]
@dotnet-bot test this please

**Pull request merged**